website/Contribution/CONTRIBUTING.rst

```diff
@@ -923,9 +923,11 @@ Models
 .. code-block:: python
 
     class Event(models.Model):
-        # Private attributes
+        # Private attributes: model declaration (_name and inherits), _description, ...
         _name = 'event.event'
+        _inherit = 'mail.thread'
```

Comment:
Yes, it's not required. But I'm not adding it for being required but to demonstrate possible order. Same as in `_order`, it will not always be provided or necessary.

Reply:
There are cases where it _is_ required. For example, [when you define a new model that inherits from a mixin](https://github.com/odoo/odoo/blob/331d8451d9011aff6a8290c473a52fa77b30b358/addons/stock/models/stock_lot.py#L22-L24), or [when you want to extend one pre-existing model with a new mixin](https://github.com/odoo/odoo/blob/331d8451d9011aff6a8290c473a52fa77b30b358/addons/sale/models/account_move.py#L8-L10).

You can fix the example with this to reduce confusion:

```suggestion
        _inherit = ['event.event', 'mail.thread']
```

Reply:
OK, good balance

Reply:
Done, suggestion applied.